{{cookiecutter.project_shortname}}/Pipfile

```diff
@@ -7,16 +7,14 @@ verify_ssl = true
 check-manifest = ">=0.25"
 
 [packages]
-invenio-app-rdm = {extras = ["{{ cookiecutter.database }}", "elasticsearch{{cookiecutter.elasticsearch}}"],version = "==1.0.0a5"}
-invenio-rdm-records = "==1.0.0a8"
-invenio-records-permissions = "==1.0.0a6"
+invenio-app-rdm = {extras = ["{{ cookiecutter.database }}", "elasticsearch{{cookiecutter.elasticsearch}}"],version = "==0.6.0"}
+invenio-rdm-records = "==0.9.0"
+invenio-records-permissions = "==0.7.0"
```

Comment:
@ppanero why do we explicitly set the `invenio-records-permissions` as it is part of the `invenio-rdm-records`?

Reply:
It is mainly to be safe in case during the next month we release something that breaks. Because people will test feb while we develop in march. So we pin the "exact" dependency each time.